src/sparseml/pytorch/optim/modifier_lr.py

```diff
@@ -273,6 +297,7 @@ def __init__(
         param_groups: Optional[List[int]] = None,
         update_frequency: float = -1.0,
         log_types: Union[str, List[str]] = ALL_TOKEN,
+        constant_logging: int = 1,
```

Comment:
constant_logging name doesn't seem to fit with what this is doing. Can we update to something like log_level